packages/git/config.mak.uname.patch

```diff
@@ -1,23 +1,24 @@
 Set uname_S to Linux instead of detecting build machine.
 Android does not support -lrt.
 
-diff -u -r ../git-2.10.0/config.mak.uname ./config.mak.uname
---- ../git-2.10.0/config.mak.uname	2016-09-02 21:59:09.000000000 -0400
-+++ ./config.mak.uname	2016-09-03 17:27:41.964893344 -0400
-@@ -1,6 +1,6 @@
- # Platform specific Makefile tweaks based on uname detection
+diff -uNr ../git-2.36.0/config.mak.uname ./config.mak.uname
+--- ../git-2.36.0/config.mak.uname	2022-04-18 02:52:38.000000000 -0300
++++ ./config.mak.uname	2022-05-03 20:43:05.835763663 -0300
+@@ -4,7 +4,7 @@
+ # Microsoft's Safe Exception Handling in libraries (such as zlib).
+ # Typically required for VS2013+/32-bit compilation on Vista+ versions.
  
 -uname_S := $(shell sh -c 'uname -s 2>/dev/null || echo not')
 +uname_S := Linux
  uname_M := $(shell sh -c 'uname -m 2>/dev/null || echo not')
  uname_O := $(shell sh -c 'uname -o 2>/dev/null || echo not')
  uname_R := $(shell sh -c 'uname -r 2>/dev/null || echo not')
-@@ -36,8 +36,6 @@
+@@ -55,8 +55,6 @@
  	HAVE_DEV_TTY = YesPlease
  	HAVE_CLOCK_GETTIME = YesPlease
  	HAVE_CLOCK_MONOTONIC = YesPlease
 -	# -lrt is needed for clock_gettime on glibc <= 2.16
 -	NEEDS_LIBRT = YesPlease
+ 	HAVE_SYNC_FILE_RANGE = YesPlease
```

Comment:
`sync_file_range` is introduced in Android API 26. I suppose this won't work.

Reply:
Damn. I'll patch it out then for now